src/utils/network/session-core.js

```diff
@@ -460,14 +450,14 @@ export const getInboxMessageTTL = async (session, inboxList) => {
  * saves a copy on your inbox
  */
 export const sendMessageTTL = async (session, messageObject) => {
-  const { title, message, recipientUsername } = messageObject;
+  const { recipientUsername } = messageObject;
   const containerUrl = getContainerUrl(
     session,
     'Inbox',
     INTERACTION_TYPES.CROSS,
     recipientUsername
   );
-  const inboxUrl = getContainerUrl(session, 'Inbox', INTERACTION_TYPES.SELF);
+  const outboxUrl = getContainerUrl(session, 'Outbox', INTERACTION_TYPES.SELF);
 
   const senderUsername = session.info.webId.split('profile')[0].split('/')[2].split('.')[0];
   const recipientWebId = `https://${recipientUsername}.${
```

Comment:
The send function breaks on Community Solid Server because of this webId generation. Just pass in the Pod URL.

Reply:
Got it. Gonna pass in PodUrl directly into the sendMessageTTL function.